cmd/ocm/account/users/cmd.go

```diff
@@ -55,6 +62,12 @@ func init() {
 		"", // Default value gets assigned later as connection is needed.
 		"Organization identifier. Defaults to the organization of the current user.",
 	)
+	flags.StringSliceVar(
+		&args.roles,
+		"roles",
+		[]string{},
+		"Role identifiers. Returns users with one or more of the specified roles. Multiple roles can be specified like: --roles=\"role1,role2,role2\".",
```

Comment:
@rhdedgar The tests are failing because this line is very long. Change this so that the line is at most 100 characters and I will then merge.

Reply:
Ok, this is now split into two lines (each < 100 characters). Thank you